### ISSUE TITLE
Error page after posting a question

### ISSUE THREAD

**Issue by blancomaberino** [q2a/question2answer#732]
I've just installed this locally (PHP 7.2/MySql 5.6.38).
After I installed it and created the first user (the admin one, following the steps after installation), I try to post a question from an incognito window (meaning I am not logged into the platform) and it shows an error page with the message `A question2answer database query failed when generating this page` despite the question is properly saved on the DB.

This is my error log:
```
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Notice:  Array to string conversion in /Applications/MAMP/htdocs/question2answer.local/qa-src/Database/DbConnection.php on line 223
[21-Jan-2019 17:18:01 America/Montevideo] PHP Question2Answer MySQL query error 42000: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''Array','Array','Array','Array','Array','Array','Array','Array'' at line 1 - Query: INSERT IGNORE INTO qa_words (word) VALUES ?,?,?,?,?,?,?,?
```

If I go to the file where it's trying to cast Array to String and dump the variables to screen, I see them all empty (https://www.dropbox.com/s/83i2ilj2tfnukxg/Screenshot%202019-01-21%2017.22.45.png?dl=0).

Any clues about what may be happening here?

Thank you.

**Comment by blancomaberino** [comment]
That seemed to be the issue. I moved to master and everything works fine there. I didn't realize `dev` was the default branch, my bad. Thank you for your quick response!

**Comment by pupi1985** [comment]
This should be fixed by #737
